crates/uv/tests/edit.rs

```diff
@@ -186,7 +186,7 @@ fn add_git() -> Result<()> {
     Installed 2 packages in [TIME]
      - project==0.1.0 (from file://[TEMP_DIR]/)
      + project==0.1.0 (from file://[TEMP_DIR]/)
-     + uv-public-pypackage==0.1.0 (from git+https://github.com/astral-test/uv-public-pypackage@0dacfd662c64cb4ceb16e6cf65a157a8b715b979?tag=0.0.1#0dacfd662c64cb4ceb16e6cf65a157a8b715b979)
+     + uv-public-pypackage==0.1.0 (from git+https://github.com/astral-test/uv-public-pypackage@0dacfd662c64cb4ceb16e6cf65a157a8b715b979)
```

Comment:
@charliermarsh Other than preference, I think this was the main issue preventing me from merging this PR. I'm not sure that this is correct.

(I do kinda prefer the URL approach since it's consistent with what we do in other places.)

Reply:
Ok, let's run with URL. It seems fine to me. Feel free to close.